### ISSUE TITLE
Bluetooth 5 support??

### ISSUE THREAD

**Issue by ordas** [randdusing/cordova-plugin-bluetoothle#563]
Hi

Please, does the plugin support Bluetooth5?

thanks!

**Comment by randdusing** [comment]
I don't believe Android or iOS offer any new APIs specific to Bluetooth 5, so there's nothing to change on the plugin side... yet.
